boranga/frontend/boranga/src/components/internal/routes/index.js

```diff
@@ -81,11 +81,11 @@ export default
             path: 'payment',
             component: ParkEntryFeesDashboard,
         },
-        {
+       /* {
             path: 'payment_order',
             component: PaymentOrder,
             name:"payment_order"
-        },
+        },*/
```

Comment:
Remove?

Reply:
Same creating issue for npm start. So I have commented for now.